variables.tf

```diff
@@ -27,3 +27,38 @@ variable "infrastructure_logging_bucket_retention" {
   description = "Retention in days for the infrasrtucture S3 logs. This is for the default S3 logs bucket, where all AWS service logs will be delivered"
   type        = number
 }
+
+variable "infrastructure_vpc" {
```

Comment:
Are we still going to have VPCs per environment?  (We need to segregate staging from prod)
This VPC will be where things shared by all infrastrucures are?

Reply:
Yep each infrastructure will be it's own environment
When using dalmatian-tools, we specify the infrastructure and environment name
https://github.com/dxw/dalmatian-tools/blob/main/bin/deploy/infrastructure